package.json

```diff
@@ -140,6 +141,7 @@
     "rollup-plugin-uglify": "6.0.4",
     "shelljs": "0.8.5",
     "shipjs": "0.26.0",
+    "stream-http": "^3.1.0",
```

Comment:
what's this needed for?

Reply:
this is also added by parcel while adding @algolia/client-composition, I'll clean this up when removing it